### ISSUE TITLE
[BUG] skip to main content links are no longer visible

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#6456]
**Describe the bug**
Skip to main content links at the top of every page that should become visible / active the first time a user hits the TAB to navigate into a page with keyboard are no longer visible.

**To Reproduce**
Steps to reproduce the behavior, including the results:
1. Go to https://avalon-dev.dlib.indiana.edu/media_objects/g158bh28
2. Hit TAB
3. The focus does not advance to the Avalon logo, but you can't see what's focused because the skip to main content link did not become visible.

**Expected behavior**
The skip to main content link should become visible when it receives active focus from keyboard navigation.

**Screenshots/Screen Recording**
Here is what is currently looks like on avalon-staging:

<img width="1121" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNjdHQ2c9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7474313fcc29fa927e3ea94ab35d7404008baee8/image.png" alt="image.png" />

**Environment (please complete the following information):**
 - Device / platform: laptop
 - OS: macOS
 - Browser: Chrome
- [x] avalon-dev
- [ ] avalon-staging
- [ ] mco-staging
- [ ] mco-production
- [ ] demo

**Release:**
8.1 regression, likely due to Bootstrap 5 changes.

**Comment by joncameron** [comment]
Should be a change in CSS to make sure it gains visibility on focus.

**Comment by masaball** [comment]
Prior bootstrap versions required `class='sr-only sr-only-focusable'` for an element to be hidden by default but visible on focus. This was changed in BS5 so that an element only requires `class='visually-hidden-focusable'` but that was missed in the upgrade, so the skip to main content was updated to use `class='visually-hidden visually-hidden-focusable'`. Apparently, `.visually-hidden` overrides `.visually-hidden-focusable`.

Removing `.visually-hidden` from the element should resolve this.
